src/assets/css/librarybrowser.scss

```diff
@@ -627,6 +622,11 @@
     }
 }
 
+.layout-mobile .detailPageContent {
+    padding-left: 5%;
+    padding-right: 5%;
+}
+
```

Comment:
Since it works.
```suggestion

        .layout-mobile & {
            padding-left: 5%;
            padding-right: 5%;
        }
    }
}

```

Reply:
Just implemented your suggestions. The .layout-mobile move wouldn't work without !important. Let me know if you have any other suggestions.

Reply:
That's becasue you have moved this block before the LTR/RTL paddings. In my suggestion, it goes after them.

Reply:
Fixed.